ard/cost/orbit_wrap.py

```diff
@@ -60,6 +62,24 @@ def generate_orbit_location_from_graph(
     edges_inclsub = [edge for edge in edges_to_process if edge[0] < 0 or edge[1] < 0]
     edges_inclsub.sort(key=lambda x: (x[0], x[1]))
 
+    # check to see if any nodes appear more than twice
+    # (i.e. once destination and possibly one source)
+    node_countmap = dict.fromkeys(
+        list(set(node for edge in edges_to_process for node in edge)), 0
+    )
+    for edge in edges_to_process:
+        node_countmap[edge[0]] += 1
+        node_countmap[edge[1]] += 1
+    if np.any(np.array(list(node_countmap.values())) > 2):
+        if allow_branching_approximation:
+            warnings.warn(
+                "The provided collection system design graph includes branching, "
+                "which ORBIT does not support. Proceeding with an approximate "
+                "radial collection system for cost modeling."
+            )
+        else:
+            raise ValueError("The graph has branching. ORBIT does not support this.")
```

Comment:
May be worth extending this warning message to indicate the option of the approximation approach and how to allow it. Not a blocking suggestion.